web/server/codechecker_server/api/report_server.py

```diff
@@ -623,11 +623,23 @@ def get_report_details(session, report_ids):
         extended_data.filePath = file_path
         extended_data_list[report_id].append(extended_data)
 
+    # Get Comments for report data
+    comment_data_list = defaultdict(list)
+    comment_query = session.query(Comment, Report.id)\
+        .filter(Report.id.in_(report_ids)) \
+        .outerjoin(Report, Report.bug_id == Comment.bug_hash)
```

Comment:
What do you think to sort the comment in an ascending order by the creation date? This way the first comment will always be the latest one.
```suggestion
        .outerjoin(Report, Report.bug_id == Comment.bug_hash) \
        .order_by(Comment.created_at.desc())
```

Reply:
There you go.. 😄